src/data.ts

```diff
@@ -1,31 +1,131 @@
+      "Stage",
+      "The Stage is an event-wide virtual stage. This is where you will see presentations and keynotes with other attendees. You will be able to see live broadcasts and some pre-recorded content here. This will replicate the real conference experience.",
+      "Sessions",
+      "Sessions promote real-time live interactions. They replicate the round tables, rooms and panels experience. Up to 20 attendees can be on one screen with hundreds of attendees watching and chatting. There is no limit to the amount of sessions that can run at the same time, it's a perfect place for round tables, group talks, and project-related discussions.",
+      "Networking",
+      "Networking allows to get to know people at CUSEC! Expand your network with ticket type-based matching and have engaging 1:1 meetings to connect with other students. This area will be used for speed networking and a microphone/webcam is recommended. Those with a VIP ticket will get priority networking in this area.",
+      "Expo",
+      "The Expo area will be used for our virtual career fair. Attendees can “walk around” to visit the booths that interest them and interact with the sponsors. Each booth at will have commercials, branded content, special offers, and some representatives on live camera.",
+    ],
+  },
+  {
+    question: "Can I use Hopin on mobile?",
+    answer: [
+      "The events that run on Hopin platform also can be accessed by Attendees and have some limited compatibility for Speakers and Organizers.",
+      "There is no dedicated app for desktops or mobiles on the Stores, no downloads are required.",
+    ],
+  },
+  {
+    question: "How much tickets cost and where do you get from them?",
+    answer: [
+      "CUSEC will be offering two ticket tiers this year.",
+      "The free tier will include access to all of our talks, events and networking sessions on Hopin.",
+      "VIP tickets are available for $25 CAD that offer extra benefits and exclusive CUSEC 2021 merch.",
+      "You can register for CUSEC on our website.",
+    ],
+  },
+  {
+    question: "What is included in a ticket?",
+    answer: [
+      "The Free ticket gives you access to the full conference.",
+      "The VIP ticket gives you extra benefits like priority networking, VIP exclusive events and CUSEC 2021 merchandise.",
+    ],
+  },
+  {
+    question: "Is there swag this year?",
+    answer: [
+      "A digital swag bag with goodies from our sponsors will be available to all our attendees this year.",
+      "Exclusive CUSEC 2021 merch will be available to those with VIP tickets, but we will have amazing prizes to give out at our events if you have a free ticket.",
+    ],
+  },
+  {
+    question: "How long do ticket sales last?",
+    answer: [
+      "Ticket sales will for VIP tickets will end earlier so we can make sure you get your merchandise in time for the conference.",
```

Comment:
Extra "will" in "Ticket sales **will** for VIP tickets will end earlier ..." (see bold)